dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -176,6 +178,7 @@ def __init__(
       self._pipeline = serialize_pipeline(pipeline)
       self._batch_size = batch_size
       self._num_threads = num_threads
+      device_id = device_id or types.CPU_ONLY_DEVICE_ID
```

Comment:
This doesn't allow to use `0` as device_id, as it will always set the second part of or statement.

Reply:
```suggestion
      device_id = device_id if device_id is not None else types.CPU_ONLY_DEVICE_ID
```
Is this the intended outcome?

Reply:
Right. I tested how it work only for None and non-0 value.

Reply:
I will keep the old way as it is more readable than one line combo.
```
if device_id is None:
  device_id = types.CPU_ONLY_DEVICE_ID
```